frontend/pages/hosts/details/HostDetailsPage/modals/CancelActivityModal/helpers.ts

```diff
@@ -0,0 +1,7 @@
+import { getErrorReason } from "interfaces/errors";
```

Comment:
Wait, what is the point in wrapping getErrorReason with a new function if it's not doing any transformations?

Reply:
Oh is it for `// TODO: hook up error message when API is updated`?

Reply:
yes; the error messaging rending will get a bit more complicated when the API is ready so I've preemptively abstracted it out